contracts/service-registry/src/msg.rs

```diff
@@ -65,6 +66,12 @@ pub enum QueryMsg {
         chain_name: ChainName,
     },
 
+    #[returns(HashSet<ChainName>)]
+    GetRegisteredChainsForWorker {
+        service_name: String,
+        worker_address: Addr,
+    },
```

Comment:
Why do we need this?

Reply:
also, we can't accept `Addr` in user input, because it's not validated. We should accept string, and then validate (`deps.api.addr_validate(...)`)

Reply:
I am using it for testing (test.rs on service registry does not have access to state.rs contents). Any other ideas?